test/stripe_test.rb

```diff
@@ -33,4 +33,21 @@ class StripeTest < Test::Unit::TestCase
       Stripe.max_network_retries = old
     end
   end
+
+  should "pass the stripe-account in the header" do
+    response = make_account(
+      charges_enabled: false,
+      details_submitted: false,
+      email: "test+bindings@stripe.com"
+    )
+    Stripe.stripe_account = 'acct_1234'
+
+    Stripe.expects(:execute_request_with_rescues).with(
+      has_entry(:headers, has_entry(:stripe_account, 'acct_1234')),
+      any_parameters,
+      any_parameters
+    ).returns(make_response(response))
```

Comment:
Hey @Edouard-chin, so `execute_request_with_rescues` is at least to some degree what I'd consider a private-level method, and it would be nice to avoid referencing it in tests if possible.

`test_helper` defines a standardized way of mocking out HTTP requests; would you mind changing this check over to use that? [Here is an example of a mock that would be very close to the one that you want to build](https://github.com/stripe/stripe-ruby/blob/c98f555aeb649ba4c42450f2bc99e77f595a8776/test/stripe/account_test.rb#L27-L31).